doc/Overview/Mvux/Advanced/InspectGeneratedCode.md

```diff
@@ -0,0 +1,26 @@
+---
+uid: Overview.Mvux.Advanced.InspectGeneratedCode
+---
+
+# Inspecting the generated code
+
+MVUX makes extensive use of code generation. It's used to generate Bindable Proxy Models and entities and commands.  
+Inspecting the generated code can give you a lot of insight into how the Bindable Proxies work under the hood and can also improve your knowledge about MVUX operates behind the scenes and familiarize yourself with its internals.
```

Comment:
Are "Bindable Proxies" the same as "Bindable Proxy Models" (from the previous sentence)?
Why different terms? WIll the reader know the difference (or that they're the same)?

This sentence reads like it should be at least two (maybe three) separate ones. 

Is there a missing **how** in the middle of "knowledge about MVUX operates"?

Is "and familiarize yourself with its internals." needed? I'm not clear about what this is trying to add.